components/generic/ContentCard.vue

```diff
@@ -76,6 +84,10 @@
       isRelated: {
         type: Boolean,
         default: false
+      },
+      thumbnail: {
```

Comment:
Can we just reuse the existing `imageUrl` prop in here and also then get rid of the new `b-img` element? If so, let's.

Reply:
Do you think where possible we should be using images instead of background images?

Reply:
Perhaps but let's consider that at a later date.